dc-cudami-editor/src/components/PublicationDatesForm.jsx

```diff
@@ -0,0 +1,79 @@
+import React, {Component} from 'react'
+import {FormGroup, Label, Row, Col} from 'reactstrap'
+import DatePicker from 'react-date-picker'
+import {useTranslation} from 'react-i18next'
+
+const PublicationDatesForm = (props) => {
+  const {t} = useTranslation()
+
+  return (
+    <FormGroup>
+      <Row>
+        <Col>
+          <Label className="font-weight-bold" for="publication-start-date">
+            {t('startPublicationDate')}:
+          </Label>
+        </Col>
+        <Col>
+          <DatePicker
+            calendarAriaLabel={t('datePickerAriaLabel.calendarAriaLabel')}
+            clearAriaLabel={t('datePickerAriaLabel.clearAriaLabel')}
+            dayAriaLabel={t('datePickerAriaLabel.dayAriaLabel')}
+            monthAriaLabel={t('datePickerAriaLabel.monthAriaLabel')}
+            nativeInputAriaLabel={t('datePickerAriaLabel.nativeInputAriaLabel')}
+            onChange={(date) =>
+              props.onChange(
+                'publicationStart',
+                date
+                  ? date.getFullYear() +
```

Comment:
- Ist das nicht das ISO-Format ohne Uhrzeit? Wenn ja, wäre `date.toISOString().slice(0, 10)` deutlich kürzer :upside_down_face: 
- Statt dem Ternary kannst du hier einfach Optional-Chaining nehmen: `date?.toISOString().slice(0, 10)`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Habe ich probiert, es gab dann einen Offset von -1 beim Tag.

Reply:
Du meinst beim Monat? Dann würde ich das Datum außerhalb vom String-Building modifizieren. Ist das nicht auch ein Bug dann in dem Date-Picker?

Reply:
Monat war korrekt, Tag wurde immer um eins nach hinten im Kalender dann angezeigt.

Reply:
Ok, weil im Code hast du Monat inkrementiert :see_no_evil:

Reply:
Genau, bei der aktuellen Lösung muss man Monat erhöhen, bei der `toISOString` den Tag, leider!

Reply:
Vielleicht kannst du die Datumsformatierung noch in eine separate Funktion auslagern.

Reply:
Ist ausgelagert.

`.toISOString()` kann verwendet werden, wenn man einen kleinen +12 Stunden Offset Workarund verwendet. Ohne diesen Workaround: Ausgewähltes Datum wird korrekt übernommen. Beim nochmaligen Laden wird allerdings der vorherige Tag angezeigt. Wenn man vor dem Speichern aber +12 Stunden drauf addiert, funktioiert es.

`